planner/core/point_get_plan.go

```diff
@@ -1940,6 +2015,10 @@ func getColumnPosInIndex(idx *model.IndexInfo, colName *model.CIStr) int {
 			return i
 		}
 	}
+
+	if idx.Global {
+		return -1
```

Comment:
Could we use a named constant instead of the anonymous `-1` to clarify what it means?

Reply:
resolved